### PR TITLE
Premine error - updates

### PR DESCRIPTION
**Pull request by leogaggl**
1) Change the number of affected hotspots (as per checks done by @gradoj )
2) Clarify the wording on Helium Foundation funding as suggested by @capjbadger007
3) Minor spelling changes

### CODE REVIEW

File: 0092-premine-error-correction.md [hunk 1]
```diff
@@ -23,16 +23,16 @@ This is to correct the errors made during the calculation and payment of the IOT
...
 
 1. A multi-sig premine correction wallet will be created to fund awarding qualifying Hotspots that were not awarded during the IOT pre-mine.
-2. Each qualifying Hotspot will be awarded 5,798.755355 (2,500,000,000 available IOT tokens / 431,127 rewarded Hotspots at migration) IOT tokens out of the multi-sig wallet and transferred to the wallet that owned the Hotspot at the time of the original pre-mine award (20th April, 2023). The specifics of how this is implemented will be 
-3. The pre-mine correction multi-sig wallet will be able to be funded with donations from any source that chooses to add to the fund over a period of 14 epochs. Any difference required for the full payment will be funded by the Helium Foundation. Any extra leftover funds will be burnt, and the wallet's keys will be disposed of.
+2. Each qualifying Hotspot will be awarded 5,798.755355 (2,500,000,000 available IOT tokens / 431,127 rewarded Hotspots at migration) IOT tokens out of the multi-sig wallet and transferred to the wallet that owned the Hotspot at the time of the original pre-mine award (20th April 2023). The specifics of how this is implemented will be
```
[the remark this answers is not in the document]
Comment (abhay) [reply 0092-premine-error-correction.md:31]: added a specific team who is responsible to make this actually happen on chain.

**Review by abhay — CHANGES_REQUESTED** [review]
one minor edit and this lgtm.

File: 0092-premine-error-correction.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (leogaggl) [reply 0092-premine-error-correction.md:31]: Thanks for those changes. 

Makes sense from my point of view!

**Review by abhay — APPROVED** [review]

**Comment by abhay** [comment]
Merging edits on behalf of author.

**Pull request merged**